recipes/pbzip2/build.sh

```diff
@@ -0,0 +1,2 @@
+make PREFIX=$PREFIX LDFLAGS="-L$PREFIX/lib $LDFLAGS" CXXFLAGS="-I$PREFIX/include $CXXFLAGS"
```

Comment:
I don't see a test suite in the source package, and there are no obvious make targets. This motivated the functional test in 6e0519a. Am I missing something?